dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/CompanyManagerTest.kt

```diff
@@ -47,10 +48,12 @@ class CompanyManagerTest(
 
```

Comment:
Why is the upload of the ``testCompanyList` done before each test, and not once before all tests?

Reply:
Dunno. Gone.

Reply:
👍